roomsrv/init_network.go

```diff
@@ -27,7 +28,12 @@ func (s *Server) initNetwork() error {
 			return &s.master, nil
 		}
 
-		if _, err := s.authorizer.GetByFeed(s.rootCtx, *remote); err == nil {
+		pm, err := s.Config.GetPrivacyMode(nil)
+		if err != nil {
+			return nil, fmt.Errorf("running with unknown privacy mode")
+		}
+		// if privacy mode is open, allow all connections. otherwise, only allow members
+		if _, err := s.authorizer.GetByFeed(s.rootCtx, *remote); err == nil || pm != roomdb.ModeOpen {
```

Comment:
I'm not 100% sure this is correct, only in restricted mode it should be restricted to members. cc @staltz 

In community mode, everyone can connect (and use aliases to connect to people directly) but just members can see endpoints.

I'd do 

```go
if pm == roomdb.ModeRestricted {
  if _, err := s.authorizer.GetByFeed(s.rootCtx, *remote); err!=nil {
    return nil, fmt.Errorf("access restricted to members")  
  }
}
```


( fyi: `s.authorizer == members list` )

Reply:
> In community mode, everyone can connect (and use aliases to connect to people directly) but just members can see endpoints.

correct

Reply:
ah alright, gotcha :)

Reply:
97cef32